ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/route53/provision/cucushift-installer-rehearse-aws-ipi-route53-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-route53-provision
+  steps:
+    - ref: ipi-conf
+    - ref: aws-provision-vpc-shared
+    - ref: aws-provision-bastionhost
+    - ref: ipi-conf-aws
```

Comment:
Suggest to move `ipi-conf-aws` behind `ipi-conf-aws-route53` step.

Reply:
Done